test/integration/targets/postgresql/tasks/postgresql_facts.yml

```diff
@@ -0,0 +1,101 @@
+# Test code for the postgresql_facts module
+# Copyright: (c) 2019, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: "postgresql_facts - test return values"
+  become_user: "{{ pg_user }}"
+  become: True
+  postgresql_facts:
+    db: "{{ db_default }}"
+    login_user: "{{ pg_user }}"
+  register: result
+  ignore_errors: True
+
+- assert:
+    that:
+      - 'result.ansible_facts.version != ""'
+      - 'result.ansible_facts.databases.{{ db_default }}.collate'
+      - 'result.ansible_facts.languages'
+      - 'result.ansible_facts.namespaces.information_schema'
+      - 'result.ansible_facts.settings'
+
+- name: "postgresql_facts - check incl_subset param passed by list"
+  become_user: "{{ pg_user }}"
+  become: True
+  postgresql_facts:
+    db: "{{ db_default }}"
+    login_user: "{{ pg_user }}"
+    incl_subset:
+      - ver*
+      - rol*
+  register: result
+  ignore_errors: True
```

Comment:
Just FYI, for documentation and examples we prefer booleans as yes/no, but we do also allow true/false.
We do not want to promote True/False or other YAML 1.1 boolean values to keep the documentation and examples consistent.

For integration tests, we do not enforce any standard.